src/AzureExtension/DevBox/DevBoxInstance.cs

```diff
@@ -128,14 +178,20 @@ private async Task<(Uri? WebURI, Uri? RdpURI)> GetRemoteLaunchURIsAsync(Uri boxU
     }
 
     public ComputeSystemOperations SupportedOperations =>
-        ComputeSystemOperations.Start | ComputeSystemOperations.ShutDown | ComputeSystemOperations.Delete;
+        ComputeSystemOperations.Start | ComputeSystemOperations.ShutDown | ComputeSystemOperations.Delete | ComputeSystemOperations.Restart;

```

Comment:
can we add ComputeSystemOperations.ApplyConfiguration as well ? If not they won't show up in the machine configuration page

Reply:
Let's add it when we actually also add the feature.